Makefile

```diff
@@ -73,11 +72,11 @@ make.addons.table:
 
 .PHONY: dispatch-test-install-upgrade
 dispatch-test-install-upgrade:
-	@{ \
-	echo "INFO: the following test groups will be run:" ;\
-	KBA_KUBECONFIG=/workspace/kba-git-src/kubeconfig ./test/dispatch-ci.sh ;\
-	cd ./test && go run -tags experimental ./scripts/test-wrapper.go ;\
-	for g in $(cd ./test && go run -tags experimental ./scripts/test-wrapper.go) ; do \
-	    shell cd ./test && go test -tags experimental -timeout 60m -race -v -run $g ; \
-	done ;\
-	}
+	cd test; ./dispatch_test_install_upgrade.sh $(KBA_BRANCH)
```

Comment:
Let's try to use either pushd, popd or absolute path.
Here the target will leave us in a different directory.